includes/Core/Util/Migration_1_123_0.php

```diff
@@ -0,0 +1,212 @@
+<?php
+/**
+ * Migration for 1.123.0
```

Comment:
Arguably this should use the `n.e.x.t` placeholder, but seeing as the class name uses the underscore-separated format with the version hardwired, I think we can keep it as-is.